solutions/standard/variables.tf

```diff
@@ -11,78 +11,99 @@ variable "region" {
+  description = "The CRN of existing secrets manager private certificate to use to create VPN."
   default     = null
 }
 
-variable "existing_sm_instance_region" {
+variable "cert_common_name" {
   type        = string
-  description = "Required if value is passed into `var.existing_sm_instance_guid`."
+  description = "A fully qualified domain name or host domain name for the certificate to be created.  Only used when `existing_secrets_manager_cert_crn` is `null`."
   default     = null
 }
 
 variable "root_ca_name" {
   type        = string
-  description = "The name of the Root CA to create for a private_cert secret engine. Only used when `var.existing_sm_instance_guid` is `false`."
-  default     = "root-ca"
+  description = "The name of the Root CA to create for a private_cert secret engine. Only used when `existing_secrets_manager_cert_crn` is `null`."
+  default     = null
 }
 
 variable "root_ca_common_name" {
   type        = string
-  description = "A fully qualified domain name or host domain name for the certificate to be created."
-  default     = "example.com"
+  description = "A fully qualified domain name or host domain name for the certificate to be created.  Only used when `existing_secrets_manager_cert_crn` is `null`."
+  default     = null
 }
 
 variable "intermediate_ca_name" {
   type        = string
-  description = "The name of the Intermediate CA to create for a private_cert secret engine. Only used when `var.existing_sm_instance_guid` is `false`."
-  default     = "intermediate-ca"
+  description = "The name of the Intermediate CA to create for a private_cert secret engine. Only used when `existing_secrets_manager_cert_crn` is `null`."
+  default     = null
 }
 
 variable "certificate_template_name" {
   type        = string
-  description = "The name of the Certificate Template to create for a private_cert secret engine. When `var.existing_sm_instance_guid` is `true`, then it has to be the existing template name that exists in the private cert engine."
-  default     = "my-template"
+  description = "The name of the Certificate Template to create for a private_cert secret engine. When `existing_secrets_manager_cert_crn` is `null`, then it has to be the existing template name that exists in the private cert engine."
+  default     = null
 }
 
-variable "cert_common_name" {
+##############################################################################
+# client-to-site VPN
+##############################################################################
+
+variable "existing_subnet_names" {
+  description = "Optionally pass a list of existing subnet names (supports a maximum of 2) to use for the client-to-site VPN. If no subnets passed, new subnets will be created using the CIDR ranges specified in the `var.vpn_subnet_cidr_zone_1` and `var.vpn_subnet_cidr_zone_2` variables."
```

Comment:
Don't use `var.` in DA variable descriptions - they don't make sense in projects :) 
Instead say "the `vpn_subnet_cidr_zone_1` input variable"

Reply:
updated